frontend/src/components/RoundedBox.tsx

```diff
@@ -0,0 +1,20 @@
+import { FormEventHandler, ReactNode } from "react";
+
+interface Props {
+  className?: string;
+  children: ReactNode;
+  onSubmit?: FormEventHandler<HTMLFormElement>;
+}
+
+export default function RoundedBox(props: Props) {
```

Comment:
Dit is niet een RoundedBox (wat ook met 1 tailwind-klasse bereikt kan worden), maar specifiek een PetitionFormBox of zoiets. Ik zou de content van de petition form ook gewoon hierin gooien om het simpel te maken en het een passende naam geven

Reply:
je zei juist dat ik de box moest extracten naar een nieuwe component.

Reply:
Dat is ook prima. Enkel de naamgeving ervan